packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputList.tsx

```diff
@@ -109,6 +109,33 @@ const OutputList = ({
         return undefined;
     };
 
+    const analytics = useAnalytics();
+    const reportTransactionCreatedToAnalytics = (
```

Comment:
nit pick, optional:

```suggestion
    const reportTransactionCreated = (
```

RBY